### DISCUSSION TITLE
How to solve (unknown error) when trying to queue prompt?

### DISCUSSION THREAD

**Discussion by catcrategames1** [comfyanonymous/ComfyUI#1617] [Q&A]
Hi guys! Sorry if I sound newbie, because I am.
I've downloaded ComfyUI, installed some models, and everytime I try to queue a prompt, all I got is (UNKNOWN ERROR).
What may be causing that?
Thanks!

**Comment by pjbrunet** [comment]
This solved it for me... run the server with `--enable-cors-header`
Explanation here... https://github.com/comfyanonymous/ComfyUI/pull/413

The only way to see the errors is by pressing F12 in the browser, they don't show up on the server side.
